metricbeat/module/ceph/mgr_cluster_disk/manifest.yml

```diff
@@ -0,0 +1,26 @@
+default: true
+input:
+  module: http
+  metricset: json
+  defaults:
+    method: POST
+    namespace: "json_namespace"
+    basepath: "/request"
+    headers:
+      Content-Type: application/json
+    query:
+      wait: "1"
+    body: '{"prefix": "df", "format": "json"}'
```

Comment:
We are collecting this information now from two metricsets, one for disks (`cluster_disk`), and another one for pools (`pool_disk`), they both make the same query (to `/api/v0.1/df`), but collect different data from the response.

We could keep this model and have two metricsets for the same data, this would allow a more intuitive transition from the old metricsets to the new ones. It would also allow to create by now the replacement for `cluster_disk`, that gets the stats for the disks and drops the `pools` array. And we can continue thinking on how to improve the http json metricset to support this kind of arrays.

Reply:
Looking at the response objects once again, I came to the conclusion that there is too much wiring, removing and transforming stuff around to have a nice looking metric structure. I'm ok with rewriting this to standard metric sets.